src/Microsoft.Identity.Client/ITokenCache.cs

```diff
@@ -75,6 +75,18 @@ public interface ITokenCache
         /// <param name="unifiedState"></param>
         void Deserialize(byte[] unifiedState);
 
+        /// <summary>
+        /// Serializes to the V3 unified cache format.
```

Comment:
Serializes the token cache to the MSAL.NET 3.x unified cache format, which is compatible with other
MSAL desktop versions (MSAL.Python and MSAL.Java for instance).

If you need to maintain SSO between an application using MSAL 2.x and this application using MSAL 3.x, you might also want to serialize with Serialize and Deserialize, otherwise just use SerializeV3 and DeserializeV2

Reply:
#resolved   fixed "DeserializeV2 -> DeserializeV3"